packages/client/src/__tests__/sendEmail.test.ts

```diff
@@ -159,4 +163,51 @@ describe("SendEmail", () =>
         extendedBookingsDate: "2021-01-31",
       },
     },
-  ]));
+  ]);
+
+  test("should use 'emailNameFrom' (if available in org setup) as 'from' attribute", async () => {
+    const additionalSetup = { ...emailSetup, emailNameFrom: "Eisbuk Team" };
+    const { organization } = await setUpOrganization({
+      ...orgSetup,
+      additionalSetup,
+    });
+
+    const emailPayload = {
+      ...saul,
+      organization,
+      type: ClientMessageType.SendBookingsLink,
+      bookingsLink: "https://eisbuk.com/bookings",
+    };
+
+    const {
+      data: { success, deliveryDocumentPath },
+    } = await httpsCallable<
+      unknown,
+      {
+        deliveryDocumentPath: string;
+        success: boolean;
+      }
+    >(
+      functions,
+      CloudFunction.SendEmail
+    )(emailPayload);
+
+    expect(success).toEqual(true);
+
+    // Check the process document
+    const deliveryDocPath = deliveryDocumentPath;
+    const deliveryDoc = await adminDb.doc(deliveryDocPath).get();
+    const deliveryDocData = deliveryDoc.data();
+
+    expect(deliveryDocData).toEqual(
+      expect.objectContaining({
+        payload: expect.objectContaining({
+          // Using email name from instead of email from
+          from: "Eisbuk Team",
```

Comment:
That's not a vali email address

Reply:
I assume the previous comment fix will address this too, right?

Reply:
Yeah. I wanted to add that for clarity